src/lib/viewers/controls/controls-root/ControlsRoot.tsx

```diff
@@ -67,7 +82,32 @@ export default class ControlsRoot {
         this.controlsEl.classList.remove('bp-is-hidden');
     }
 
+    shouldForceShow(): boolean {
+        if (this.wasClosedByUser.tooltipFlowAnnotationsExperience) {
```

Comment:
Can we move this logic to another component? ControlsRoot is currently pretty generic and shouldn't know about what controls/experiments it's rendering. I'm thinking something like:

```
<ControlsLayer onMount={this.handleMount}>
    <ControlsExperiments onClosed={this.helpers.reset()}>
        {controls}
    </ControlsExperiments>
</ControlsLayer>